bin/verify_boilerplate.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
```

Comment:
We still use bazel/repo-infra inside this repository, so I think we can use their boilerplate verification.

Reply:
can we just not use bazel for the verify tasks?

wdty @justaugustus ?

Reply:
adding the bazel to make all the verify checks

Reply:
ok, now we need to update the job again to have the bazel installed, which image should I use for that?

Reply:
I'm aiming at removing bazel in this repo, so I'd prefer the checks to be in bash/python, please. You can use the ones we have in k/release.

Reply:
cool, thanks @justaugustus

Reply:
was more easy just to drop the bazel commits :)